server/src/main/java/org/elasticsearch/common/lucene/Lucene.java

```diff
@@ -808,6 +809,10 @@ public void delete() {
         }
     }
 
+    public static String getCommitId(SegmentInfos segmentInfos) {
```

Comment:
can this get some javadocs?

Reply:
Sure, added in https://github.com/elastic/elasticsearch/pull/63963/commits/193b631a5c6f48e6a12beb1e0e6b04fee97c5fb9.